Contributors/guide/IDEA.md

```diff
@@ -1,100 +1,141 @@
 # Setup StarRocks FE development environment on IDEA
 
-## Requirements
-
-### thrift compiler
+This tutorial is based on macOS and has been tested on Apple Chip(M1, M2).
+Even if you are not using macOS, you can also refer to this tutorial.
 
-Now, StarRocks needs thrift 0.13.x. So you need to install thrift-0.13.0 first.
+## Requirements
 
-#### Mac
+**Thrift 0.13**
 
-On Mac you can install it with the following steps.
+If you installed Thrift directly with brew, you will find that there is no version 0.13, you can use the following command:
 
-```
+```bash
 brew tap-new $USER/local-tap
 brew extract --version='0.13.0' thrift $USER/local-tap
 brew install thrift@0.13.0
 ```
-After installing thrift successfully, you can check by executing following command in command-line.
 
-```
+After installing Thrift successfully, you can check by executing following command:
+
+```bash
 $ thrift -version
```

Comment:
I think the fastest way to compile successfully on idea is to modify https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/pom.xml#L65